scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/ParquetAvroFileOperations.java

```diff
@@ -83,8 +83,13 @@ private ParquetAvroFileOperations(
 
   public static <V extends GenericRecord> ParquetAvroFileOperations<V> of(
       Schema schema, FilterPredicate predicate) {
+    return of(schema, predicate, new Configuration());
```

Comment:
The same way we have a protected `DEFAULT_COMPRESSION` could we have one for the config? We can use it every making things consistent.